js/app.js

```diff
@@ -1 +1,291 @@
-console.log('Hello frontend');
+document.addEventListener('DOMContentLoaded', function() {
+	console.log('loaded up the DOM');
+
+var x = "X";
+var o = "O";
+var turn = 2;
+
+var t1 = document.getElementById('t1');
+var t2 = document.getElementById('t2');
+var t3 = document.getElementById('t3');
+var t4 = document.getElementById('t4');
+var t5 = document.getElementById('t5');
+var t6 = document.getElementById('t6');
+var t7 = document.getElementById('t7');
+var t8 = document.getElementById('t8');
+var t9 = document.getElementById('t9');
+
+var gBoard = {
+	sq1: null,
+	sq2: null,
+	sq3: null,
+	sq4: null,
+	sq5: null,
+	sq6: null,
+	sq7: null,
+	sq8: null,
+	sq9: null,
+}
+
+var newGame = document.getElementById('reset');
+
+
+
+	t1.addEventListener('click', function(event) {
```

Comment:
You can define this function and reuse it instead of repeating the code inline everytime. The parameter passed by the click event is the item that got clicked.

```
t1.addEventListener("click", clickSquare);

function clickSquare(e){
    // e refers to the element that was clicked.
    // You can use in place of selecting element by id, etc.
}
```